lib/wrapper.js

```diff
@@ -399,6 +432,20 @@ class RMLMapperWrapper {
       };
     });
   }
+
+  async _cleanUpStateFolder() {
+    const stateSubFolders = await fsTraditional.promises.readdir(this.stateFolder);
```

Comment:
Why does the code use `fsTraditional.promises` when that is exactly what `fs-extra` does?

Reply:
I remember trying hard to obtain the same using `fs-extra`, but couldn't get it to work. `fsTraditional.promises.readdir(this.stateFolder)` returns a `string[]`, while `fs-extra` uses callbacks to do the processing... Probably my javascript skills are too limited; do you have an idea how to do the same with `fs-extra`?

Reply:
If I do a local test then `await fsTraditional.promises.readdir` and `await fs.readdir` give the same results.

Reply:
Aah, now I see: if no callback is passed, a promise with the result will be returned :upside_down_face: . See, need to improve my JS knowledge. But it works indeed.